website/docs/patterns/data-visualization/partials/guidelines/color-usage.md

```diff
@@ -0,0 +1,46 @@
+## Color usage
+
+When creating data visualizations, care should be taken to select colors geared towards the type of data being presented. General color palettes created for web UIs and/or marketing purposes are not always suitable as the colors can lack contrast and distinguishability or else be visually unsuitable when used together in a data context. Keep in mind that color choice can also impact meaning.
```

Comment:
> Keep in mind that color choice can also impact meaning.

I agree with this. Could there be any scenarios in which someone may want to use the HDS palette in a chart to indicate success, warning, or error since we've already defined those colors for those use cases?

Reply:
There very well could be. I don't know of specific examples currently but that's something we could think about.